ansible_collections/arista/avd/roles/eos_cli_config_gen/README.md

```diff
@@ -1773,6 +1773,15 @@ router_ospf:
       max_lsa: < integer >
       default_information_originate:
         always: true
+      summary:
```

Comment:
change to `summary_address` to match cli.

Reply:
Fixed by commit 8b7c7977